README.md

```diff
@@ -67,6 +67,9 @@ To learn more about this library you can explore the following additional resour
 * [ASP.NET Core samples](https://github.com/okta/samples-aspnetcore)
 * [Project Documentation](https://github.com/okta/okta-aspnet/tree/master/docs)
 
+You can find an example of implementation of refresh token middleware [here](https://github.com/okta/okta-aspnet/blob/master/docs/refresh-token.md)
```

Comment:
I think this link should be formatted similarly to the others in the list above it.  Also, I think the url should probably use "tree" instead of "blob" for consistency; not completely sure the consequences one way or the other.